projects/clr/hipamd/src/hip_texture.cpp

```diff
@@ -1468,6 +1468,7 @@ hipError_t hipTexRefSetMipmappedArray(textureReference* texRef, hipMipmappedArra
   resDesc.res.mipmap.mipmap = mipmappedArray;
 
   hipTextureDesc texDesc = hip::getTextureDesc(texRef);
+  texDesc.normalizedCoords = 1;
```

Comment:
you can't just force normalized coords you need to check the flag
#define CU_TRSF_NORMALIZED_COORDINATES 0x02